### PR TITLE
updates imports for v12 and wires new version bundle

### PR DESCRIPTION
**Pull request by xh3b4sd**
Follow up of https://github.com/giantswarm/kvm-operator/pull/478. Successfully tested on `geckon`.

### CODE REVIEW

File: service/controller/cluster.go [hunk 1]
```diff
@@ -303,6 +322,7 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
 				resourceSetV9,
 				resourceSetV10,
 				resourceSetV11,
+				resourceSetV12,
```
Comment (xh3b4sd) [inline service/controller/cluster.go]: Wiring the cluster controller.

File: service/controller/drainer.go [hunk 2]
```diff
@@ -106,13 +107,30 @@ func newDrainerResourceRouter(config DrainerConfig) (*controller.ResourceRouter,
...
+	}
+
 	var resourceRouter *controller.ResourceRouter
 	{
 		c := controller.ResourceRouterConfig{
 			Logger: config.Logger,
 
 			ResourceSets: []*controller.ResourceSet{
 				resourceSetV11,
+				resourceSetV12,
```
Comment (xh3b4sd) [inline service/controller/drainer.go]: Wiring the drainer controller.

File: service/controller/v12/cloudconfig/cloudconfigtest/cloudconfigtest.go [hunk 3]
```diff
@@ -3,7 +3,7 @@ package cloudconfigtest
 import (
 	"github.com/giantswarm/micrologger/microloggertest"
 
-	"github.com/giantswarm/kvm-operator/service/controller/v11/cloudconfig"
+	"github.com/giantswarm/kvm-operator/service/controller/v12/cloudconfig"
```
Comment (xh3b4sd) [inline service/controller/v12/cloudconfig/cloudconfigtest/cloudconfigtest.go]: Updating all the imports.

File: service/controller/v12/cluster_resource_set.go [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package v11
+package v12
```
Comment (xh3b4sd) [inline service/controller/v12/cluster_resource_set.go]: Updating all the packages.

File: service/controller/v12/drainer_resource_set.go [hunk 5]
```diff
@@ -32,22 +32,13 @@ func NewDrainerResourceSet(config DrainerResourceSetConfig) (*controller.Resourc
...
-		//	return false
-		//}
-
-		// TODO this check has to be enabled when going from v11 to v12.
-		//if v == VersionBundle().Version {
-		//	return true
-		//}
+		if v == VersionBundle().Version {
+			return true
+		}
```
Comment (xh3b4sd) [inline service/controller/v12/drainer_resource_set.go]: This bit here is important for the routing which I wanted to fix finally.

File: service/controller/v12/resource/clusterrolebinding/resource.go [hunk 6]
```diff
@@ -11,7 +11,7 @@ import (
 
 const (
 	// Name is the identifier of the resource.
-	Name = "clusterrolebindingv11"
+	Name = "clusterrolebindingv12"
```
Comment (xh3b4sd) [inline service/controller/v12/resource/clusterrolebinding/resource.go]: Updating all the resource names.

File: service/controller/v12/version_bundle.go [hunk 7]
```diff
@@ -123,8 +48,8 @@ func VersionBundle() versionbundle.Bundle {
 		Dependencies: []versionbundle.Dependency{},
 		Deprecated:   false,
 		Name:         "kvm-operator",
-		Time:         time.Date(2018, time.April, 25, 13, 00, 0, 0, time.UTC),
-		Version:      "2.2.0",
-		WIP:          false,
+		Time:         time.Date(2018, time.May, 18, 15, 00, 0, 0, time.UTC),
+		Version:      "2.2.1",
+		WIP:          true,
```
Comment (xh3b4sd) [inline service/controller/v12/version_bundle.go]: Updating the new version bundle.

File: service/version_bundles.go [hunk 8]
```diff
@@ -28,6 +29,7 @@ func NewVersionBundles() []versionbundle.Bundle {
 	versionBundles = append(versionBundles, v9.VersionBundle())
 	versionBundles = append(versionBundles, v10.VersionBundle())
 	versionBundles = append(versionBundles, v11.VersionBundle())
+	versionBundles = append(versionBundles, v12.VersionBundle())
```
Comment (xh3b4sd) [inline service/version_bundles.go]: Wiring the version bundle.

**Review by fgimenez — APPROVED** [review]
LGTM

**Pull request merged**